Flush logs when running non-interactive
Patch author: Alexander Inyukhin <shurick@sectorb.msk.ru>

See http://bugs.debian.org/691069

--- a/src/digitemp.c
+++ b/src/digitemp.c
@@ -587,6 +587,7 @@
     close( fd );
   } else {
     printf( line );
+    fflush( stdout );
   }
   return 0;
 }  
